packages/react/src/components/InlineLoading/InlineLoading.js

```diff
@@ -41,8 +48,7 @@ export default function InlineLoading({
           className={`${prefix}--inline-loading__checkmark-container`}
         />
       );
-    }
-    if (status === 'inactive' || status === 'active') {
+    } else if (status === 'inactive' || status === 'active') {
```

Comment:
since the `if` block contains a return statement the `else` is unneeded

```suggestion
    } 
    if (status === 'inactive' || status === 'active') {
```